engine/src/main/java/org/terasology/rendering/dag/stateChanges/DisableDepthWriting.java

```diff
@@ -61,61 +55,26 @@ public StateChange getDefaultInstance() {
     }
 
     @Override
-    public RenderPipelineTask generateTask() {
-        if (enabled) {
-            return enablingTask;
-        } else {
-            return disablingTask;
-        }
-    }
-
-    @Override
     public boolean equals(Object obj) {
-        return (obj instanceof DisableDepthWriting) && this.enabled == ((DisableDepthWriting) obj).isEnabled();
+        return (obj instanceof DisableDepthWriting) && this.enabled == ((DisableDepthWriting) obj).enabled;
     }
 
     @Override
     public int hashCode() {
         return Objects.hashCode(enabled);
     }
 
-    public boolean isEnabled() {
-        return enabled;
-    }
-
-    public String getStatus() {
-        String status = "disabled";
-        if (enabled) {
-            status = "enabled";
-        }
-
-        return status;
+    private String getStatus() {
+        return enabled? "Enabled": "Disabled";
```

Comment:
The reason I removed this from here is, the isEnabled() you talk about will be a generic Node/StateChange funciton, not a DisableDepthWriting StateChange function.